ds_judgements_public_ui/sass/includes/_search.scss

```diff
@@ -49,30 +52,33 @@
   }
 
   &__more-options-beta {
-    margin-top: calc($spacer-unit / 2);
+    margin-top: -0.75rem;
```

Comment:
Can we try to avoid negative margins. Let's reduce the spacing above this rather than adding negative margin here.